package.json

```diff
@@ -25,8 +25,7 @@
     "cheerio": "^1.0.0-rc.10",
     "entities": "^5.0.0",
     "figlet": "^1.7.0",
-    "postman-request": "^2.88.1-postman.34",
-    "signalr-client": "0.0.20"
```

Comment:
OK - was used in `onNotification` but is replaced as of https://github.com/noblox/noblox.js/commit/d626e6f49d504f3a2fbfa3cda85b999beee02cc5